test_helpers/shared_feature_context.go

```diff
@@ -61,11 +61,26 @@ func SharedFeatureContext(s *godog.Suite) {
 
 	// Application Setup
 
+	s.Step(`^I am in the root directory of a non-exosphere application called "([^"]*)"$`, func(appName string) error {
+		appDir = path.Join(os.TempDir(), appName)
+		if err := util.CreateEmptyDirectory(appDir); err != nil {
+			return errors.Wrap(err, fmt.Sprintf("Failed to create an empty %s directory", appDir))
+		}
+		return nil
+	})
+
 	s.Step(`^I am in the root directory of an empty application called "([^"]*)"$`, func(appName string) error {
 		appDir = path.Join(os.TempDir(), appName)
 		return createEmptyApp(appName, cwd)
 	})
 
+	s.Step(`^it doesn\'t run any tests$`, func() error {
+		if childCmdPlus != nil {
+			return childCmdPlus.WaitForText("Not an application or service directory, exiting...", time.Minute)
+		}
+		return validateTextContains(childOutput, "Not an application or service directory, exiting...")
```

Comment:
Thoughts on having a variable for holding the expectedText to reduce the duplication?